model/host/db.go

```diff
@@ -548,6 +548,11 @@ func ById(id string) db.Q {
 	return db.Query(bson.D{{Key: IdKey, Value: id}})
 }
 
+// ByIP produces a query that returns a host with the given ip address.
+func ByIP(ip string) db.Q {
+	return db.Query(bson.D{{Key: IPKey, Value: ip}})
```

Comment:
I realize the ById function is written with a bson.D but that's actually pretty outdated syntax; we usually stick to `bson.M{<key>: <value>}` these days, it's a bit more readable

Reply:
done